plugin/api.thrift

```diff
@@ -130,6 +130,24 @@ struct Argument {
      * Argument type.
      */
     2: required Type type
+    /**
+     * Annotations defined on this argument.
+     *
+     * Given,
+     *
+     *   void setValue(
+     *   	1: SetValueRequest req
+     *   ) throws (
+     *		1: BadRequestError badRequestError (cache = "false")
+     *   )
```

Comment:
indentation is a bit off here with tabs/spaces. for the thrift file, we should use spaces only.

Reply:
using spaces only now